README.md

```diff
@@ -1,169 +1,79 @@
 # kubeadm-nspawn
 
-## Introduction
-
 __kubeadm-nspawn__ is a tool for creating a multi-node Kubernetes cluster
 on a single machine, created mostly for developers __of__ Kubernetes.
 
 It aims to be as similar to the solutions recommended for production
 clusters as possible.
 
-## Getting started
-
-### Build proper systemd-nspawn version
-
-Unfortunately, there is [one pending feature](https://github.com/systemd/systemd/pull/4395)
-of systemd-nspawn which is merged in master, but not released yet.
-You will need to build your own systemd-nspawn binary.
-
-We highly recommend using CoreOS' fork which backported that feature
-to the 231 version of systemd (which is the one that Fedora and
-the other popular distributions are using in its stable releases).
-
-In order to do that, please use the following commands:
-
-```
-git clone git@github.com:coreos/systemd.git
-cd systemd
-git checkout v231
-./autogen.sh
-./configure
-make
-```
-
-You **shouldn't** do `make install` after that! Using the custom
-systemd-nspawn binary with the other components of systemd being
-in another version is totally fine.
-
-You may try to use master branch from upstream systemd repository, but
-it's very risky!
+## Requirements
 
-### Get needed Kubernetes repositories
+* **Host:** (tested with Fedora 26)
+  - systemd-nspawn v233
 
-kubeadm-nspawn needs the following repos to exist in your GOPATH:
+* **Container:** (tested with CoreOS Alpha)
+  - docker v17.05
```

Comment:
Maybe I should not put it there at all and document the subcommands instead. This information can go there.